src/contractTest/resources/application-pact.yaml

```diff
@@ -0,0 +1,38 @@
+spring:
+  cloud:
+    openfeign:
+      client:
+        config:
+          default:
+            loggerLevel: none
+  jpa:
+    hibernate:
+      ddl-auto: none
+    database-platform: org.hibernate.dialect.PostgreSQLDialect
+    properties:
+      hibernate:
+        boot:
+          allow_jdbc_metadata_access: false
+  sql:
+    init:
+      mode: never
+
+idam:
+  api:
+    url: ${IDAM_API_URL:http://localhost:8891}
+
+logging:
+  level:
+    root: info
+    au.com.dius: info
+    uk.gov.hmcts.reform: DEBUG
+
+pact:
+  idam:
+    redirect_uri: http://www.dummy-pact-service.com/callback
+    grant_type: password
+    client_id: laupact
+    client_secret: pactsecret
+    username: lau-pact-user@hmcts.net
+    password: Password123
```

Comment:
Can you replace the "Password123" with something that will not be picked up by Fortify as false positive.

Reply:
I believe no matter what we add, since it has property named "password", it will be picked up. And I imagine the same for `client_secret`